docs/runbooks/DeprecatedRHEL6Vm.md

```diff
@@ -0,0 +1,22 @@
+# DeprecatedRHEL6Vm
+
+## Meaning
+
+This alert fires when a running virtual machine is based on RHEL6 template, which will not be supported in the next release.
+
+## Impact
+RHEL6 virtual machines will not be supported in the next release in case of issues.
+
+## Diagnosis
+
+1. Check which running virtual machines has the rhel6 template label.
+
+   ```bash
+   $ kubectl get -A vm -ojson | jq  '.items | map(select((.metadata.labels."vm.kubevirt.io/template" | test("rhel6-.*")?) and .status.printableStatus == "Running")) | {kind: "List", items: .}'  | kubectl get -A -f -
+   ```
+
+## Mitigation
+
+Upgrade all the RHEL6 virtual machines to a newer version or stop them.
```

Comment:
@fossedihelm 
Should they also update the template metadata to indicate that the VMI is not RHRL6 anymore?

Reply:
Done, thanks